packages/nimble-components/src/label-provider/rich-text/index.ts

```diff
@@ -5,7 +5,8 @@ import {
     richTextToggleBoldLabel,
```

Comment:
FYI restored [users/aagash/mention-in-editor](https://github.com/ni/nimble/tree/users/aagash/mention-in-editor) and retargeted this branch to it to make reviewing a bit easier. But should merge from main for final reviews.

Reply:
We did the below steps:

1. Merged the angular changes [PR ](https://github.com/ni/nimble/pull/1686)to the `main` branch.
2. Merged the latest `main` branch to this PR branch. (This brings this PR on top of latest `main` branch)
3. Updated this PR's target branch to the `main` branch.